packages/vue/src/components/authenticator.vue

```diff
@@ -193,6 +193,12 @@ watch(
     signOut.value = s;
   }
 );
+
+const hasTabs = () => {
+  return (
+    actorState.value?.matches('signIn') || actorState.value?.matches('signUp')
+  );
+};
```

Comment:
This works, but it's more performant if we switch this to a `computed` property. It won't run nearly as often, then a method since `computed` properties are cached. 

```suggestion
const hasTabs = computed(() => {
  return (
    actorState.value?.matches('signIn') || actorState.value?.matches('signUp')
  );
});
```

Make sure to then change the `hasTabs()` to `hasTabs` without the parenthesis. 

Later on, I should add in `useAuthenticator` and grab the data from `route` instead of `actorState`. But I can do that later.

Reply:
Ah, TIL! Thanks, will adjust.

Reply:
Done! 54197f3